language_generators/scala-defref-extractor/src/main/scala/io/bazeltools/buildgen/shared/DriverApplication.scala

```diff
@@ -107,6 +123,8 @@ abstract class DriverApplication extends IOApp {
     ).mapN {
       (inputs, workingDirectory, label_or_repo_path, outputPath, sequential) =>
         for {
+          specialTlds <- getEnv("BZL_GEN_SPECIAL_TLDS")
+          _ <- setSpecialTlds(specialTlds.getOrElse(""))
```

Comment:
Why not: `specialTlds.traverse_(setSpecialTlds)`

Might also fix the test errors.

Reply:
I tried this but actually I think it's also worth treating the empty string similarly, so I ultimately used a different approach.